views/Geographic/GeographicMap.svelte

```diff
@@ -13,20 +17,44 @@
   export let contentWidth;
   export let baseMap;
   export let formattingOptions;
+  export let maxHeight = 550;
```

Comment:
Why are you explicitly setting the `maxHeight` here again? Since you're passing down the property from the Choropleth-component, I dont think you have to do this?